docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md

```diff
@@ -11,12 +11,110 @@ menu:
 
-If you are using Yugabyte Platform to manage universes, you need to call the following REST API endpoint on your Yugabyte Platform instance for the producer universe and the consumer universe involved in the asynchronous replication between two data sources:
+Within the concept of replication, universes are divided into the following categories:
+
+- A source universe contains the original data that is subject to replication.
+
+  Note that in the current release, replicating a source universe that has already been populated with data can be done only by contacting Yugabyte Support.
+
+- A target universe is the recepient of the replicated data. One source universe can replicate to one or more target universes.
+
+For additional information on asynchronous replication in YugabyteDB, see [Asynchronous Replication: Overview and Architecture](https://docs.yugabyte.com/latest/architecture/docdb-replication/async-replication/).
+
+## Use the Yugabyte Platform UI
+
+You can use the Yugabyte Platform UI to set up and configure asynchronous replication. In addition, you can perform monitoring by accessing the information about the replication lag and enabling alerts on excessive lag.
+
+### How to set up replication
+
+You can set up asynchronous replication for a universe which does not contain data as follows:
+
+1. Open the Yugabyte Platform UI and navigate to **Universes**.
+
+2. Select the universe you want to replicate and navigate to **Replication**.
+
+3. Click **Configure Replication** to open the dialog shown in the following illustration:<br><br>
+
+   ![Configure Replication](/images/yp/asynch-replication-2.png)<br><br>
+
+4. Provide the name for your replication.
+
+5. Select the target universe.
+
+6. Click **Next: Select Tables**.
+
+7. From a list of common tables between source and target universes, select the tables you want to include in the replication and then click **Create Replication**, as per the following illustration:<br><br>
+
+   ![Create Replication](/images/yp/asynch-replication-3.png)
+
+### How to configure replication
+
+You can configure an existing replication as follows:
+
+1. Open the Yugabyte Platform UI and navigate to **Universes**.
+
+2. Select the universe whose existing replication you want to modify and then navigate to **Replication**, as per the following illustration:<br><br>
+
+   ![Replication](/images/yp/asynch-replication-1.png)<br><br>
+
+3. Click **Configure Replication** and perform steps 4 through 7 from [How to set up replication](#set-up).
+
+### How to view, manage, and monitor replication
+
+To view and manage an existing replication, as well as configure monitoring, click the replication name to open the details page shown the following illustration:
+
+<br>
+
+![Replication Details](/images/yp/asynch-replication-4.png)
+
+This page allows you to do the following:
+
+- View the replication details.
+
+- View and modify the list of tables included in the replication, as follows:
+
```

Comment:
Any constraints (see comment above)?

Reply:
Modified the intro sentence to cover the empty tables constraint: 
"You can use the Yugabyte Platform UI to set up and configure asynchronous replication for universes whose tables do not contain data. "